net-ftp/uftpd/metadata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
```

Comment:
Please also add the Proxy-maint project to the metadata.xml file. Members of this project will proxy your changes.

Reply:
@monsieurp done!